src/kirby/components/toast/config/toast-config.ts

```diff
@@ -0,0 +1,7 @@
+import { ThemeColor } from './../../../helpers/theme-color.type';
+
+export interface ToastConfig {
+  message?: string;
+  duration?: number;
```

Comment:
Rename to make it more clear:
```suggestion
  durationInMs?: number;
```